.happy/terraform/modules/sfn_config/nextstrain-dbg.wdl

```diff
@@ -9,7 +9,7 @@ workflow nextstrain {
         String remote_dev_prefix = ""
         String group_name
         String s3_filestem
-        Map[String, String] template_args
+        String template_args
```

Comment:
We might be sending structures with mixed string or int values, and fighting with MiniWDL's typing system for (semi-arbitrary) structures sounds like a nightmare. `template_args` will just be a json-encoded string from now on.